src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -432,9 +433,86 @@
                     </ControlTemplate.Triggers>
                 </ControlTemplate>
             </Setter.Value>
-
         </Setter>
     </Style>
+    <!-- CustomGridView related styles -->
+    <Style TargetType="GridViewColumnHeader" x:Key="GvchNoContent">
+        <Setter Property="Margin" Value="0,0,0,2"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type GridViewColumnHeader}">
+                    <Border Background="{DynamicResource ResourceKey=ColumnHeaderBGBrush}" Height="20"/>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+    <Style TargetType="Thumb" x:Key="ThumbGridViewColumnHeader">
+        <Setter Property="HorizontalAlignment" Value="Right"/>
+        <Setter Property="Width" Value="3"/>
+        <Setter Property="Margin" Value="0,0,3,0"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type Thumb}">
+                    <Border Padding="{TemplateBinding Padding}" Width="3" Background="Transparent">
+                        <Rectangle HorizontalAlignment="Center" Width="3" Fill="Transparent"/>
+                    </Border>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+    <Style TargetType="Border" x:Key="BrdrDataGridCell">
+        <Setter Property="Height" Value="20"/>
+        <Setter Property="Margin" Value="-4,0"/>
+        <Setter Property="Background" Value="{DynamicResource ResourceKey=DataGridBGBrush}"/>
+        <Style.Triggers>
+            <DataTrigger Binding="{Binding RelativeSource={RelativeSource AncestorType=ListViewItem,Mode=FindAncestor},Path=IsSelected}" Value="True">
+                <Setter Property="Background" Value="{DynamicResource ResourceKey=DataGridSelectedBGBrush}"/>
+            </DataTrigger>
+        </Style.Triggers>
+    </Style>
+    <Style TargetType="TextBlock" x:Key="TbDataGridCellContent" BasedOn="{StaticResource VarStandardTextBlock}">
+        <Setter Property="Foreground" Value="{DynamicResource ResourceKey=TextBrush}"/>
+        <Setter Property="HorizontalAlignment" Value="Left"/>
+        <Setter Property="Margin" Value="4,0"/>
+        <Setter Property="VerticalAlignment" Value="Center"/>
+        <Style.Triggers>
+            <DataTrigger Binding="{Binding RelativeSource={RelativeSource Mode=Self}, Path=Text}" Value="{x:Static properties:Resources.PropertyDoesNotExist}">
```

Comment:
Do we still have access to the vm bound value here (e.g. using null instead of PropertyDoesNotExist) or has the transformation already happened?

Reply:
This style applies to all columns, so we can't count on the VM property existing